plugin/configuration.py

```diff
@@ -73,6 +74,33 @@ def _on_done(self, index):
             self.window.status_message("{} enabled, starting server...".format(config_name))
 
 
+class LspExecuteCommand(sublime_plugin.WindowCommand):
```

Comment:
You might want to inherit from LspTextCommand (located in plugin/core/registry.py). This handles disabling the command if the view does not support LSP.